__tests__/3-get-api.test.js

```diff
@@ -0,0 +1,20 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+
+
+
+describe("get/api/", () => {
+    test("get 200, return an object describing all the available endpoints on your API", () => {
+        return request(app).get("/api").expect(200)
+            .then((res) => {
+                expect(res.type).toBe('application/json')
+                expect(res.body.content).toHaveProperty('GET /api')
+                expect(res.body.content).toHaveProperty('GET /api/topics')
+                expect(res.body.content).toHaveProperty('GET /api/articles')
+               
```

Comment:
Nice job, remember if you stick with this method you will need to update this for each endpoint.
You could require in the endpoints.json and assert that it looks the same as your request.